### DISCUSSION TITLE
In which country is the Supreme People’s Assembly?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#341] [Galaxy Trivia] [answered]
**Hint:** 🇰🇵

**Comment by GameExplorer** [comment, marked as answer]
North Korea

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/GameExplorer?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and keep the great questions posting to the commentary.. 😊🎉

**Comment by thuongtruong1009** [comment]
The Supreme People's Assembly is the highest organ of state power in North Korea, and it is located in the capital city of Pyongyang. The Supreme People's Assembly is the unicameral legislative body of North Korea and is composed of deputies elected from each of the country's constituencies. The Assembly is responsible for enacting and amending the country's constitution, electing the country's leaders, and overseeing the work of the government and other state organs.
